api/admin/problem_details.py

```diff
@@ -102,9 +102,23 @@
 
 LIBRARY_NOT_FOUND = pd(
     "http://librarysimplified.org/terms/problem/library-not-found",
+    status_code=404,
+    title=_("Library not found"),
+    detail=_("The specified library uuid does not exist."),
```

Comment:
I didn't notice this before, but can you combine the admin LIBRARY_NOT_FOUND and the api LIBRARY_NOT_FOUND? Not a big deal if that's annoying, since they use the same URI.